editor/src/components/inspector/sections/component-section/variables-in-scope-utils.ts

```diff
@@ -224,32 +225,54 @@ function variableInfoFromVariableData(variableNamesInScope: VariableData): Array
   return info
 }
 
+function keepOnlyFirstMatch(variables: Array<VariableInfo>): Array<VariableInfo> {
+  let foundFirstMatch: boolean = false
+  let result: Array<VariableInfo> = []
+  for (const variable of variables) {
+    const matches = foundFirstMatch ? false : variable.matches
+    result.push({ ...variable, matches })
+    if (variable.matches) {
```

Comment:
I think it would be more readable to just check `matches` instead of `variable.matches` (and maybe move it right after declaring it)

Reply:
The suggestion makes sense, let's implement it as a follow-up improvement (for posterity: at the time of writing we were under time pressure to merge this PR)